src/schema/types/models/Portfolio.ts

```diff
@@ -63,22 +63,15 @@ export const Portfolio = objectType({
 export const PortfolioSkill = objectType({
   name: "PortfolioSkill",
   definition(t) {
-    t.field("skill", {
-      type: "Skill",
-      resolve: () => {
-        return "Typescript or something";
-      },
-    });
+    t.string("name");
```

Comment:
기존의 Skill type을 두어야 skill table에 있는 정보를 이용해서 사용할 수 있을거에요. gql로 불러올 때는 
```
skill {
  name
}
```
처럼 불러오면 돼요:D
```suggestion
    t.field("skill", {
      type: "Skill",
    });
```

Reply:
일반적인 스키마가 아닌, 넥서스 오브젝트 타입이기에 그렇게 작성할 경우 null만 반환하더라고요

Reply:
제가 실수를 했었네요, 죄송합니다.
코멘트 내용을 반영하여 커밋하였습니다.